package.json

```diff
@@ -74,7 +74,7 @@
   },
   "lint-staged": {
     "*.{js,jsx}": [
-      "eslint"
+      "eslint --fix"
```

Comment:
I don' think this will work without also adding `git add` after it, because it modifies files. Is there a need to do this `--fix` here or can you run `eslint --fix` manually when needed?